apps/crn-server/test/data-providers/contentful/users.data-provider.test.ts

```diff
@@ -51,7 +51,35 @@ describe('User data provider', () => {
           contentfulRestClientMock,
         );
       const result = await userDataProviderWithMockServer.fetchById('123');
-      const expectation = getUserDataObject();
+      const expectation = {
+        ...getUserDataObject(),
+        workingGroups: [
```

Comment:
looks like this is part of the standard return fixture now, why don't you just extend the USerDataObject fixture then so you don't have to repeat it all the time?

Reply:
i think i did thay before and ended up breaking at lot of test as this fixture is used across different entities.